backend/app/api/src/endpoints/global/members/PatchOrganizationMembersEndpoint.ts

```diff
@@ -455,6 +460,17 @@ export class PatchOrganizationMembersEndpoint extends Endpoint<Params, Query, Bo
                     });
                 }
 
+                const isEnabled = membershipType.isEnabled(organizationTags, memberDefaultAgeGroupIds);
```

Comment:
Als de request niet gescoped is tot een organization (organization = null), kan je nog steeds een membership toevoegen voor organization x, en dan gaan de tags niet gecheckt worden (dat gebeurde vroeger uiteraard ook niet, maar als we het nu toch toevoegen)